src/components/send/SendAssetList.js

```diff
@@ -80,9 +80,18 @@ export default class SendAssetList extends React.Component {
       assets.pop(); // removes not needed coin divider
     }
 
+    //check for placeholder ETH & remove
+    assets[0]?.isPlaceholder ? assets.shift() : null;
     const visibleAssetsLength = assets.length;
+
     this.data = assets;
+    console.log(this.data);
+    console.log(this.data.length);
     if (smallBalances.assets.length > 0) {
+      //check for placeholder ETH & remove
+      smallBalances.asset = smallBalances.assets.filter(
+        asset => asset?.isPlaceholder
```

Comment:
did you mean !asset?.isPlaceholder (keeping anything that is not a placeholder)

Reply:
i did, it was working anyways because of the ?